content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,40 @@ menu:
 weight: 101
 ---
 
+## v2.0.7 [2021-TK]
+
+### SQLite Metadata Store
+
+This release adds an embedded SQLite database
+for storing metadata required by the latest UI features like notebooks and annotations.
+
+### Features
+- Add Geo graph type to be able to store in Dashboard cells. <!-- https://github.com/influxdata/influxdb/pull/19811:  -->
+- Add the properties of a static legend for line graphs and band plots. <!-- https://github.com/influxdata/influxdb/pull/21218:  -->
+- Remove feature flags for permanent UI features: <!-- https://github.com/influxdata/influxdb/pull/21531:  -->
+  `axisTicksGenerator`, `legendOrientation`, `mosaicGraphType`, and `bandPlotType`.
+- Allow hiding the tooltip independently of the static legend. <!-- https://github.com/influxdata/influxdb/pull/21547:  -->
+- Support pagination when listing users via the API. <!-- https://github.com/influxdata/influxdb/pull/21367:  -->
```

Comment:
Do we have a task-based list user to add sorting options?
offset, limit, after, descending